### ISSUE TITLE
Fix wdl/dev branch Makefile/dependency issues in preparation for merging into master

### ISSUE THREAD

**Issue by mewthew** [uchicago-cs/chiventure#261]
After merging our feature branches (wdl/load_rooms and wdl/load_into_structs), we realized that there are dependency issues raised by other team functions and circular dependency issues raised by our own functions, such as a function being declared in both load_rooms and load_into_structs, which we need to address before merging into master. Kate and I will be making sure that the Makefile for our tests run so that we can test our loading into game-state structs code. These changes are being done in wdl/fixing-compiler-errors.

**Comment by mewthew** [comment]
We have fixed the Makefile compile errors by adding in the libraries and library paths of both game-state and action_management. We have also fixed merge errors regarding multiple function definitions/implementation that was causing our code to fail. We are now ready to test our functions for loading rooms into game-state structs and fix any errors that arise from that. The above issue can be found at #267.
